### PR TITLE
RHMAP-15719 - Add unit tests for admin teams commands

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/RHMAP-15719

### CODE REVIEW

File: lib/common.js [hunk 1]
```diff
@@ -430,7 +430,7 @@ exports.createTableForProperties = function(type, opts, properties) {
 
 exports.createTableForTeams = function(teams) {
   var table = new Table({
-    head: ['id', 'Name'],
+    head: ['ID', 'Name'],
```
Comment (wtrocki) [inline lib/common.js]: This may broke automation scripts we have. Can we have it reverted? Is that change needed?

Comment (camilamacedo86) [inline lib/common.js]: OKs. I will revert. This change was just to follow the same way that it is another places. ( the ID label)

**Comment by camilamacedo86** [comment]
@wtrocki the change requested into the review made. Could you check it now?
Tks for the review:-)
